compiler/luci-interpreter/src/kernels/L2Pool2D.cpp

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
```

Comment:
Please add TensorFlow copyright here. Others look good.

Reply:
Okay, I will add it.

Reply:
Do we need to? It does not look like it has TFLite code

Reply:
> Do we need to?

Just for safety. I think this is a gray area.. We do not use exactly the same code, but we follow the same logics. To me, the code from line 75 to 86 look very similar to the code in TFLite (due to the same variable names and orders).